Captain-AJ.md

```diff
@@ -0,0 +1,54 @@
+# name: "@Captain-AJ"
+# project: "An autonomous butler robot"
+
+# Project Name
+A.L.F.R.E.D. Butlerbot
+
+## Summary
+
+I plan to build an Arduino controlled autonomous robot butler named A.L.F.R.E.D. (Advanced, Little, Fit, Robot, Education, Device) that can be summoned at the press of a 
+button and deliver items. I am excited because this practical project has been on my list for sometime now. 
+After a long part search, my efforts to build this contraption summed in the listed Devastator robot kit. The set includes treads, metal motors, an aluminum tank chassis, 
+and a battery compartment.
+All coincides with my desire to apply computer and mechanical engineering concepts to develop a computing module in physical excellence.
+
+## Plan
+
+1. Order kit and required parts.
+
+2. Read a sizable amount of Arduino Cookbook to become well endowed with appropriate knowledge.
+
+3. Build the kit according to the instructions: 
+a. Assemble the tracks with gear components. 
+b. Assemble the robot's chassis and attach the Romeo BLE.
+c. Attach all sensors to the body, e.g., Ultrasonic sensor, speaker, etc.
+d. Pan/tilt systems for ultrasonic sensor and camera
+e. Wire all components 
+f. Attach the food storage unit.
+
+4. Programing (Python, C language):
+a. PixyCamera for color and facial recognition.
+b. Ultrasonic sensor for object detection and avoidance.
+c. Speaker for communication.
+d. OLED display for facial expressions and personality.
+e. Gyroscope for absolute direction capabilities, e.g. field-oriented drive.
+f. Unify sensor control by programming manual and autonomous control functions.
+
+    The robot is now complete!
+
+
+| Devastator Tank Mobile Robot Platform (Metal DC Gear Motor) | https://www.robotshop.com/products/devastator-tank-mobile-robot-platform-metal-dc-gear-motor | 84.90 |
```

Comment:
i'm wondering, why are you picking a robot set/kit instead of maybe an open-sourced robot guide?

Reply:
DFRobot’s kit is ideal as it contains the basics for my idealized plan without being restraining, which would limit my creativity. Considering this idea, the pack seems to be an excellent step in my quest toward excellence in engineering knowledge, as I am learning, revamping, and upgrading the nearly barebones kit and will gain wisdom from the engineering success proffered by DFRobot’s (an affiliate of Arduino) standards.

Reply:
thanks for this additional context :)